l10n_it_ricevute_bancarie/views/configuration_view.xml

```diff
@@ -15,6 +15,7 @@
                     <field
                         name="bank_id"
                         domain="[('partner_id.ref_company_ids', 'in', [company_id])]"
+                        options="{'no_create': True}"
```

Comment:
Qui e sotto, come mai non è più possibile creare i record?
Non mi sembra una modifica che ha a che fare con il problema corretto dalla PR, è possibile rimuoverla?
Se la vuoi mantenere andrebbe fatta in un commit a parte e con issue dedicata.

Reply:
Sì, se vuoi posso fare un commit a parte.
E' collegata a questa PR perchè i quick create permettono di creare ed impostare al volo un record senza considerare l'azienda.

Reply:
Se il problema è che vengono creati record di un'altra azienda allora potrebbe essere utile impostare dei default tipo `context="{'default_company_id': company_id}"`, come fatto ad esempio in https://github.com/odoo/odoo/blob/17f3fc980bf65f3ca4a02da998afafdfd386e0de/addons/account/views/account_tax_views.xml#L141

Creare record da qui almeno a me per lo sviluppo è utile (come vedi nel video in https://github.com/OCA/l10n-italy/pull/3617#discussion_r1362344818)

Reply:
Effettivamente in test è utile, ma in produzione per i funzionali che commettono tanti typos non è il massimo.
Anche @francesco-ooops mi consigliava di evitare i quick create quando permettono di creare oggetti importanti come registri, conti e sezionali.

Reply:
ok, ho rimosso le inibizioni dei quick create e sembra che comunque non sia possibile impostare un record di un'altra azienda perchè i campi company sono `readonly`.
Forse nel modulo dei cespiti mi dava più problemi il quick create anche se come dicevo la maggior parte degli utenti con il quick create fa grossi casini